### DISCUSSION TITLE
Where Ulancher looks at the executable files? Wish to delete dead link

### DISCUSSION THREAD

**Discussion by Cruising0904** [Ulauncher/Ulauncher#1222] [Q&A] [answered]
Hi! 
I have been using Ulauncher with love from the beginning when I first start using linux. Ulauncher is great never fails. Thanks for creating and distributing awesome app. 

![Ulauncher](https://user-images.githubusercontent.com/29267295/234320475-17010520-890c-4bed-aed6-9532d312f6f1.png)
When I type Joplin there are two Joplin, one of bottom one is not executed How can I get rid of this??
Also How to know where the file is executed from? I looked up /usr/share/applications directory and ~/.local/share/applications directory there's nothing about joplin..

**Comment by friday** [comment, marked as answer]
All app launchers work the same way: https://wiki.archlinux.org/title/desktop_entries

**Reply by Cruising0904** [reply]
Hey thanks! It is really helpful. I will study this
